### PR TITLE
Owl next fix ged

### PR DESCRIPTION
**Pull request by ged-odoo**

### CODE REVIEW

File: tests/components/__snapshots__/t_set.test.ts.snap [hunk 1]
```diff
@@ -47,6 +47,150 @@ exports[`t-set slot setted value (with t-set) not accessible with t-esc 2`] = `
...
+  function slot1(ctx, node, key = \\"\\") {
+    ctx = Object.create(ctx);
+    ctx[isBoundary] = 1
+    ctx[\`v\`] = new LazyBlock(value1, ctx, node);
+    let b3 = text(\` in slot \`);
+    let b4 = safeOutput(ctx['v']);
+    return multi([b3, b4]);
+  }
+  
+  function value1(ctx, node, key = \\"\\") {
```
Comment (Polymorphe57) [inline tests/components/__snapshots__/t_set.test.ts.snap:65]: the key param should not be there

Comment (Polymorphe57) [reply tests/components/__snapshots__/t_set.test.ts.snap:65]: evaluate in LazyBlock does not pass it to value1

Comment (ged-odoo) [reply tests/components/__snapshots__/t_set.test.ts.snap:65]: yeah, it's just the way functions are compiled by the generator. it also happens for slots

File: src/app/template_helpers.ts [hunk 2]
```diff
@@ -101,6 +101,25 @@ function shallowEqual(l1: any[], l2: any[]): boolean {
   return true;
 }
 
+class LazyBlock {
```
Comment (Polymorphe57) [inline src/app/template_helpers.ts:104]: strange to call it LazyBlock: no ref to block here

**Review by Polymorphe57 — APPROVED** [review]

**Pull request merged**